api/core/helper/position_helper.py

```diff
@@ -13,18 +13,10 @@ def get_position_map(folder_path: str, *, file_name: str = "_position.yaml") ->
     :param file_name: the YAML file name, default to '_position.yaml'
     :return: a dict with name as key and index as value
     """
-    position_file_name = os.path.join(folder_path, file_name)
-    if not position_file_name or not os.path.exists(position_file_name):
-        return {}
-    
-    positions = load_yaml_file(position_file_name, ignore_error=True)
-    position_map = {}
-    index = 0
-    for _, name in enumerate(positions):
-        if name and isinstance(name, str):
-            position_map[name.strip()] = index
-            index += 1
-    return position_map
+    position_file_path = os.path.join(folder_path, file_name)
+    yaml_content = load_yaml_file(file_path=position_file_path) or []
```

Comment:
`or []` here is unnecessary.

Reply:
Here's the part of the key change in this PR.
As while a None value returned as`load_yaml_file` loads an empty YAML file (eg. all lines commented),  an default `[]` is required in place.

Reply:
Due to the original comments, I mistook this would return an empty dictionary. The revised version is better. Thank you.